src/shared/web3/types.ts

```diff
@@ -1,21 +1,22 @@
-import { TypedDataDomain, TypedDataField } from 'ethers';
 import { EIP1193Provider } from 'mipd';
+import { Chain as ViemChain } from 'viem';
 
 export type Hex = `0x${string}`;
 
-export interface Chain {
-  id: number;
-  name: string;
-  logo: string;
-  nativeCurrency: {
-    name: string;
-    symbol: string;
-    decimals: number;
-  };
-  rpcUrls: string[];
-  blockExplorerUrls: string[];
-}
-
+// export interface Chain {
```

Comment:
Do we want to keep that interface commented?

Reply:
hell no 😅